### PR TITLE
Add/event dropdown

### PR DESCRIPTION
**Pull request by KlausKidman**

### CODE REVIEW

File: src/components/UiSelect/UiSelectOptions.vue [hunk 1]
```diff
@@ -60,15 +60,15 @@ export default {
   computed: {
     filteredOptions () {
       return this.items.filter(option => option.show && !option.hidden)
-    }
+    },
```
Comment (napalmpapalam) [inline src/components/UiSelect/UiSelectOptions.vue]: remove extra commas everywhere you added

File: src/components/UiPhoto/PhotoField.vue [hunk 2]
```diff
@@ -24,7 +24,10 @@
             <i class="mdi mdi-file ui-photo-field__icon-preview" />
           </div>
 
-          <div v-if="!$attrs.disabled" class="ui-photo-field__change-preview-wrp">
+          <div
```
Comment (napalmpapalam) [inline src/components/UiPhoto/PhotoField.vue:27]: not necessary

File: src/components/UiSelect/index.vue [hunk 3]
```diff
@@ -374,6 +374,7 @@ export default {
 
       this.isOpen = true
       this.selectedIndex = this.flattenOptions.indexOf(this.currentItem)
+      this.$emit('is-dropdown-shown', this.isOpen)
```
Comment (napalmpapalam) [inline src/components/UiSelect/index.vue]: replace with on `close` and `open` events

File: src/components/UiPhoto/PhotoField.vue (hunk 2 above)
Comment (KlausKidman) [reply src/components/UiPhoto/PhotoField.vue:27]: fixed

File: src/components/UiSelect/index.vue (hunk 3 above)
Comment (KlausKidman) [reply src/components/UiSelect/index.vue]: fixed

File: src/components/UiSelect/UiSelectOptions.vue (hunk 1 above)
Comment (KlausKidman) [reply src/components/UiSelect/UiSelectOptions.vue]: fixed

**Pull request merged**